README.md

```diff
@@ -46,6 +46,17 @@ powershell
 Set-ExecutionPolicy unrestricted
 ```
 
+#### If build with PowerShell does not work
```

Comment:
This is too similar to a previous heading. Something more along the lines of "Build Errors" would fit better.

Reply:
ok I'll do that!